src/gui/PasswordEdit.h

```diff
@@ -45,9 +45,14 @@ private slots:
     void autocompletePassword(const QString& password);
 
 private:
+    void secureInputEntry(bool enabled);
     bool passwordsEqual() const;
 
     QPointer<PasswordEdit> m_basePasswordEdit;
+
+protected:
```

Comment:
Protected section before private section. See CONTRIBUTING style guide.

Reply:
Done! (This isn't actually mentioned in the style guide).

Reply:
Damn you win this time! 😆 I forgot to put a protected section in there.

https://github.com/keepassxreboot/keepassxc/blob/develop/.github/CONTRIBUTING.md#classes